cmd/juju/common/controller.go

```diff
@@ -129,6 +129,7 @@ func WaitForAgentInitialisation(
 			strings.HasSuffix(errorMessage, "connection refused"),
 			strings.HasSuffix(errorMessage, "connection is shut down"),
 			strings.HasSuffix(errorMessage, "i/o timeout"),
+			strings.HasSuffix(errorMessage, "deadline exceeded"),
```

Comment:
I definitely don't love all these error string tests, but adding to it for now...